airbyte-integrations/connectors/source-tiktok-marketing/source_tiktok_marketing/schemas/basic_reports.json

```diff
@@ -0,0 +1,142 @@
+{
+  "type": ["null", "object"],
+  "properties": {
+    "metrics": {
+      "type": ["null", "object"],
+      "properties": {
+        "campaign_name": {
+          "type": ["null", "string"]
+        },
+        "campaign_id": {
+          "type": ["null", "integer"]
+        },
+        "adgroup_name": {
+          "type": ["null", "string"]
+        },
+        "placement": {
+          "type": ["null", "string"]
+        },
+        "adgroup_id": {
+          "type": ["null", "integer"]
+        },
+        "ad_name": {
+          "type": ["null", "string"]
+        },
+        "ad_text": {
+          "type": ["null", "string"]
+        },
+        "tt_app_id": {
+          "type": ["null", "integer"]
+        },
+        "tt_app_name": {
+          "type": ["null", "string"]
+        },
+        "mobile_app_id": {
+          "type": ["null", "integer"]
+        },
+        "promotion_type": {
+          "type": ["null", "string"]
+        },
+        "dpa_target_audience_type": {
+          "type": ["null", "string"]
+        },
+        "spend": {
+          "type": ["null", "number"]
+        },
+        "cash_spend": {
+          "type": ["null", "number"]
+        },
+        "voucher_spend": {
+          "type": ["null", "number"]
+        },
+        "cpc": {
+          "type": ["null", "number"]
+        },
+        "cpm": {
+          "type": ["null", "number"]
+        },
+        "impressions": {
+          "type": ["null", "number"]
+        },
+        "clicks": {
+          "type": ["null", "number"]
```

Comment:
It seems the docs: https://business-api.tiktok.com/marketing_api/docs?rid=sta6fe2yww&id=1707957200780290
state the types to be string, can we be sure that this is the right type for these columns?

(risk of raising exceptions if we end up having records not respecting the types when emitting data...)

(moreover clicks are probably integer, not float number usually)

Reply:
One contributor checked these reports' streams with real data and all worked normally. 
Sure  reach clicks impressions should be integer. I've fixed it